tests/playbooks_manager/test_controller.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+from django.test import TransactionTestCase
+from api_app.models import Job
+from intel_owl.tasks import start_playbooks
+from api_app.serializers import PlaybookObservableAnalysisSerializer
+from tests import PollingFunction
+
+
+class PlaybooksScriptTestCase(TransactionTestCase):
+    # attrs
+    test_job : Job
+
+    def setUp(self):
+        playbooks_to_test = os.environ.get("TEST_PLAYBOOKS", "").split(",")
```

Comment:
please add this custom value in the env_file_app_template and in the Contribute.md with the other

Reply:
oop. how did i miss pushing that. i did add it to the ci file